packages/dcos-integration-test/buildinfo.json

```diff
@@ -0,0 +1,7 @@
+{
+  "requires": [
+      "dnspython",
```

Comment:
Ideally should depend on all the things used directly (Not splitting beautifulsoup4 out of dcos-image-deps is fine). Just note when people go and modify dcos-image-deps they'll get a longer dev cycle than they expect since more things end up needing to change (although that package is relatively quick to build).